src/migrations/20240222123532-project-metrics-summary-trends.js

```diff
@@ -0,0 +1,42 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+            CREATE TABLE IF NOT EXISTS public.project_client_metrics_trends
+            (
+                project varchar NOT NULL,
+                date date NOT NULL,
+                total_yes integer NOT NULL,
+                total_no integer NOT NULL,
+                total_apps integer NOT NULL,
+                total_flags integer NOT NULL,
+                total_environments integer NOT NULL,
+                PRIMARY KEY (project, date)
+            );
+
+            INSERT INTO project_client_metrics_trends (project, date, total_yes, total_no, total_apps, total_flags, total_environments)
```

Comment:
I don't think we need this insert in a migration

Reply:
Why?